proto.lock

```diff
@@ -5259,8 +5259,13 @@
                   },
                   {
                     "id": 6,
-                    "name": "deadline",
+                    "name": "deadline_21_10_0",
```

Comment:
not that important, but consider `deadline_as_date_time` since the generated name `Deadline21100` is a bit weird